AntennaTracker/Parameters.h

```diff
@@ -117,6 +117,8 @@ class Parameters {
 
         // 254,255: reserved
```

Comment:
this is the only thing was worried about here, does it matter that we are using 255 and above?

Reply:
The comment is wrong - or at least misleading :-)

The new parameters here are at 226 and 227.